src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -28,19 +28,33 @@
 
 package cloud.orbit.spring;
 
-import cloud.orbit.actors.Stage;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import cloud.orbit.actors.Stage;
+import cloud.orbit.actors.extensions.ActorExtension;
+
 import java.util.List;
 
 @ConfigurationProperties(prefix = "orbit.actors")
 public class OrbitActorsProperties {
+    private String basePackages;
     private String clusterName;
     private String nodeName;
     private Stage.StageMode stageMode;
+    private List<ActorExtension> extensions;
     private Long messagingTimeoutInMilliseconds;
     private Long timeToLiveInSeconds;
     private List<String> stickyHeaders;
+    private Integer concurrentDeactivations;
+    private Long deactivationTimeoutMillis;
```

Comment:
Is it ok that it would differ from the hk2 key name?

From Stage:
```java
@Config("orbit.actors.deactivationTimeoutMillis")
private long deactivationTimeoutMillis = TimeUnit.MINUTES.toMillis(2);
```